integration_tests/dbt_project.yml

```
@@ -8,25 +8,25 @@ profile: 'integration_tests'
 vars:
   recharge_source:
     recharge_schema: recharge_source_integration_tests_05
-    recharge_address_identifier: "address_data"
-    recharge_address_discounts_identifier: "address_discounts_data"
-    recharge_address_shipping_line_identifier: "address_shipping_line_data"
-    recharge_charge_identifier: "charge_data"
-    recharge_charge_discount_identifier: "charge_discount_data"
-    recharge_charge_line_item_identifier: "charge_line_item_data"
-    recharge_charge_order_attribute_identifier: "charge_order_attribute_data"
-    recharge_charge_shipping_line_identifier: "charge_shipping_line_data"
-    recharge_charge_tax_line_identifier: "charge_tax_line_data"
-    recharge_checkout_identifier: "checkout_data"
-    recharge_customer_identifier: "customer_data"
-    recharge_discount_identifier: "discount_data"
-    recharge_one_time_product_identifier: "one_time_product_data"
-    order: "{{ ref('order_data') }}" # Comment out for generating docs -- Uncomment this before sending to buildkite.
-    # recharge_order_identifier: "order_data" # Uncomment for generating docs -- Comment this out before sending to buildkite.
-    recharge_orders_identifier: "orders_data"
-    recharge_order_line_item_identifier: "order_line_item_data"
-    recharge_subscription_identifier: "subscription_data"
-    recharge_subscription_history_identifier: "subscription_history_data"
+    recharge_address_identifier: "address_data" # @docs-include
```

Sorry I should have explained this a bit more. You only need to add the tag if the state of the line needs to be flipped from how it exists currently. 
```suggestion
    recharge_address_identifier: "address_data"
```

---

Thanks for explaining this!